BAZProjectC1/Menu/MenuCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MenuCell.swift
+//  BAZProjectC1
+//
+//  Created by efloresco on 20/09/22.
+//
+
+import UIKit
+
+
+enum ElectionDate: String {
+    case title = "title"
+    case detail = "detail"
+    case image = "image"
+}
+class MenuCell: UITableViewCell {
```

Comment:
Cuida los espacios entre definiciones.
```suggestion
}

class MenuCell: UITableViewCell {
```